packages/sdk-core/src/modules/attribute/FileAttributeManager.ts

```diff
@@ -0,0 +1,65 @@
+import { jsonEscaper } from '../../common/jsonEscaper';
+import { BacktraceModule, BacktraceModuleBindData } from '../BacktraceModule';
+import { FileSystem, SessionFiles } from '../storage';
+import { AttributeManager } from './AttributeManager';
+
+const ATTRIBUTE_FILE_NAME = 'bt-attributes';
+
+export class FileAttributeManager implements BacktraceModule {
+    private _attributeManager?: AttributeManager;
+
+    constructor(private readonly _fileSystem: FileSystem, private _fileName?: string) {}
+
+    public static create(fileSystem: FileSystem) {
+        return new FileAttributeManager(fileSystem);
+    }
+
+    public static createFromSession(sessionFiles: SessionFiles, fileSystem: FileSystem) {
+        const fileName = sessionFiles.getFileName(ATTRIBUTE_FILE_NAME);
+        return new FileAttributeManager(fileSystem, fileName);
+    }
+
+    public initialize(): void {
+        this.saveAttributes();
+    }
+
+    public bind?({ attributeManager, sessionFiles }: BacktraceModuleBindData): void {
+        if (this._fileName) {
+            throw new Error('This instance is already bound.');
+        }
+
+        if (!sessionFiles) {
+            return;
+        }
+
+        this._fileName = sessionFiles.getFileName(ATTRIBUTE_FILE_NAME);
+        this._attributeManager = attributeManager;
+        attributeManager.attributeEvents.on('scoped-attributes-updated', () => this.saveAttributes());
+    }
+
+    public dispose?(): void {
+        this._fileName = undefined;
+    }
+
+    public async get(): Promise<Record<string, unknown>> {
```

Comment:
I think you can use AttributeType instead of unknown. 
```suggestion
    public async get(): Promise<Record<string, AttributeType>> {
```

Reply:
Done